--- libraryroot~sp.unmin.js	2022-03-23 07:02:35.376383325 -0400
+++ libraryroot~sp.unmin.patched.js	2022-03-23 07:04:08.160218572 -0400
@@ -1,3 +1,5 @@
+console.info('[SMM] Loading patched libraryroot~sp.js');
+
 /* Third-party software licenses can be found at licenses.txt */
 (window.webpackJsonp = window.webpackJsonp || []).push([
     [2], {
@@ -138569,6 +138571,8 @@
                     }, this.m_LastLibraryTab = {
                         strActiveTab: ""
                     }, this.m_bIsDeckFactoryImage = !1, this.m_bPreviouslyIsRunningApp = !1, this.m_bHomeAndQuickAccessButtonsEnabled = !0, this.m_flLastHomePressMS = 0, this.m_flLastQuickAccessPressMS = 0, this.m_bShowingGlobalModal = !1, this.m_iHideFooterCount = 0, this.m_fnRootCancelHandler = null, this.m_currentBrowserAndBackstack = null, this.m_currentBrowserIsExternal = !1, this.m_runningAppIDs = [], this.m_streamingAppID = 0, this.m_runningAppWindowID = 0, this.m_mapAppWindows = new u.b
+                    console.info('Attach OpenQuickAccessMenu to window...');
+                    window.OpenQuickAccessMenu = this.OpenQuickAccessMenu;
                 }
                 InitWithoutUser() {
                     return Object(n.a)(this, void 0, void 0, (function*() {
